<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-size: 14px;
}

QCheckBox{
color:rgb(255,255,255);
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
}

QSpinBox{
color:rgb(255,255,255);
}

QScrollArea {
background: rgb(60, 60, 60);
}

QListWidget {
border: 2px solid gray;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: rgb(60, 60, 60);
color: white;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
font-size: 22px;
}


QTextEdit {
border: 2px solid gray;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: rgb(60, 60, 60);
color: white;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
}

QLabel {
background: none;
color: white;
max-height: 30px;
}

QProgressBar {
color: white;
border: 2px solid  rgb(80, 80, 80);
text-align: center;
padding: 1px;
border-radius: 5px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
width: 30px;
height: 14px; 
}

QProgressBar::chunk {

border-radius: 3px;
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0.0 #fff, stop: 0.4999 #8f0,  stop: 0.5 #8f0, stop: 1.0 #050);
border: 1px solid rgb(0, 0, 0);
}

QGroupBox {
border: 2px solid;
border-radius: 5px;
border-color: rgb(85, 255, 0);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
padding-top: 12px;
color: white;
selection-color: rgb(30, 30, 30);
}

QPushButton {
color: white;
border: 2px solid rgb(80, 80, 80);
border-radius: 5px;
padding: -2px;

background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #ddd, stop: 1 #111);
min-width: 80px;
min-height:30px;
}

QPushButton:hover {
border: 2px solid  rgb(160, 160, 160);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #fff, stop: 1 #111);
}

 QPushButton:pressed {
color:  rgb(85, 255, 0);
border: 2px solid rgb(85, 255, 0);
background: qradialgradient(cx: 0.5, cy: -0.5,
fx: 0.5, fy: -0.5,
radius: 1.35, stop: 0 #fff, stop: 1 #111);
}





QScrollBar:horizontal {
border: 2px solid;
border-color: rgb(85,255,0);
border-radius: 5px;
background: rgb(60,60,60);
height: 25px;
margin: 2px 2px 2px 2px;
}
QScrollBar::handle:horizontal {
background: rgb(255,255,255);
min-width: 30px;
}
QScrollBar::add-line:horizontal {
background: none;
width: 20px;
subcontrol-position: right;
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
background: none;
width: 20px;
subcontrol-position: left;
subcontrol-origin: margin;
}
QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
width: 3px;
height: 3px;
background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
background: none;
}



QScrollBar:vertical {
border: 2px solid;
border-color: rgb(85,255,0);
border-radius: 5px;
background: rgb(60,60,60);
width: 25px;
margin: 2px 2px 2px 2px;
}
QScrollBar::handle:vertical {
background: rgb(255,255,255);
min-height: 30px;
}
QScrollBar::add-line:vertical {
background: none;
height: 20px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
background: none;
height: 20px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:vertical {
width: 3px;
height: 3px;
background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:vertical {
background: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">background: rgb(60, 60, 60);</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">background: rgb(60, 60, 60);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>348</width>
              <height>349</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Graph update time (sec):</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Show advanced info:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="checkBox_advanced_info">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>on</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Show CPU graph:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="checkBox_cpu_usage">
               <property name="text">
                <string>on</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Show RAM graph:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="checkBox_ram_usage">
               <property name="text">
                <string>on</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>System info:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Log:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>task starts:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>task stops:</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>save log:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QCheckBox" name="checkBox_log_task_starts">
               <property name="text">
                <string>on</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QCheckBox" name="checkBox_log_task_stops">
               <property name="text">
                <string>on</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QCheckBox" name="checkBox_save_log">
               <property name="text">
                <string>on</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_graph_update_time">
               <property name="styleSheet">
                <string notr="true">QMenu{
border: 2px solid rgb(85, 255, 0);
border-radius: 5px;
background: none;
max-width:170px;
max-height:170px;
	selection-color: rgb(85, 255, 0);
	selection-background-color: rgb(90, 90, 90);
}</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="suffix">
                <string> sec</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Menu</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_info">
           <property name="text">
            <string>info</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_back">
           <property name="text">
            <string>back</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
